lib/mix/tasks/swagger.generate.ex

```diff
@@ -133,9 +133,13 @@ defmodule Mix.Tasks.Phoenix.Swagger.Generate do
     %{
       controller: controller,
       swagger_fun: swagger_fun,
-      path: format_path(route_map.path)
+      path: format_path(path)
     }
   end
+  defp find_swagger_path_function(_route) do
+    %{controller: nil, swagger_fun: nil, path: nil}
+  end
```

Comment:
Yes, but it may be not very clear after some time. Can you please add comment about these nil(s) to code and I'll merge this.